application/src/main/scala/com/azavea/franklin/crawler/CatalogStacImport.scala

```diff
@@ -153,7 +153,7 @@ class CatalogStacImport(val catalogRoot: String) {
                       )
                     ),
                     ().asJsonObject,
-                    forItem.properties,
+                    forItem.properties.asJson.asObject.getOrElse(JsonObject.empty),
```

Comment:
Ouf, I also don't know how to handle it better.

Reply:
Yeah I spent about four seconds adding an `Encoder.AsObject` in `stac4s` but decided not to -- it's not hard but it didn't seem important enough